index.html

```diff
@@ -3519,42 +3534,93 @@ <h2>
               [=Flow content=]
             </td>
           </tr>
+          <tr>
+            <th tabindex="-1" id="index-aria-checkbox">
+              <a data-cite="wai-aria-1.2#checkbox">`checkbox`</a>
+            </th>
+            <td>
+              <a data-cite="wai-aria-1.2#aria-checked">`aria-checked`</a>
+            </td>
+            <td>
+              <ul>
+                <li>
+                  <a data-cite="wai-aria-1.1#aria-disabled">`aria-disabled`</a></li>
```

Comment:
all of these should probably also point to `wai-aria-1.2` rather than `wai-aria-1.1`